q2_feature_classifier/_skl.py

```diff
@@ -21,7 +21,8 @@
             'non_negative': True}],
           ['classify',
            {'__type__': 'custom.LowMemoryMultinomialNB',
-            'alpha': 0.01}]]]]
+            'alpha': 0.01,
+            'chunk_size': 20000}]]]]
```

Comment:
I was able to confirm that the error I mentioned in #57 no longer occurs when I run `qiime feature-classifier fit-classifier-naive-bayes` with this branch. I tested both with using the default for `--p-classify--chunk-size`, and specifying `--p-classify--chunk-size 20000`. 🎉